Assets/HoloToolkit-Examples/Input/Scripts/NavigationRotateResponder.cs

```diff
@@ -41,13 +42,13 @@ private void PerformRotation()
 
         public void OnNavigationCanceled(NavigationEventData eventData)
         {
-            navigationDelta = Vector3.zero;
+            navigationDelta = Vector3.zero; //Set it back to default values
```

Comment:
One rule of thumb I use while writing comments is that if you need to write a comment that explains what your code does, you need to write more easily understood code. Comments should explain WHY you're doing it the way you're doing it, not HOW or WHAT your code is doing.

Here's a made up example:

**Good:**
```
OnSleep()
{
NetworkObject.Reset(); // We need to reset our network object, since they can get out of sync if packets arrived when we were sleeping
}
```

**Bad:**
```
// This function gets called when the app goes to sleep.
AwakeningNetworkObjectRecovery()
{
NetworkObject.Reset();
}
```